src/main/webapp/load-facebook.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
```

Comment:
nit: can add this back to the main JS if you want

Reply:
I'll just leave it in for now.